specs/multispec.yml

```diff
@@ -0,0 +1,20 @@
+version: 1
+
+specs:
+  distroinfo:
+    fedora:
+      distros:
+        - fedora-26-x86_64
+        - fedora-25-x86_64
+      vendor: "Fedora Project"
+      source_url: "some.url.fedoraproject.org"
+      distro_specific_help: "Some Fedora specific help"
```

Comment:
hm? what's the point of doing this? this is not an example repo, this is a real image